modules/mod_authentication/templates/logon_error/message.unequal.tpl

```diff
@@ -0,0 +1,4 @@
+    <p>{_ The two passwords should be equal. Please retype them. _}</p>
+
+    <p>{_ Passwords should have at least six characters. _}<p>
```

Comment:
Doesn’t this go for `tooshort` only?

Reply:
It is a bit silly that this error is hard-coded this way, especially as the length is a configuration.
What do you propose?  Echo the minimum length here?

Reply:
Maybe just remove it and force the minimum length using a validator.

Reply:
There is already a validator:

```
            type={
                length minimum=min_length
                too_short_message=_"Your password is too short." ++ " " ++ _"Minimum characters: " ++ min_length
            }
```

So the whole message length string is not needed.  I'll remove it.